dash/linux-dash.yml

```diff
@@ -0,0 +1,11 @@
+dash:
+  image: husseingalal/busydash
+  privileged: true
+  links:
+  - network
+  labels:
+  - io.rancher.os.scope=system
+  restart: always
+  pid: host
+  ipc: host
+  net: host
```

Comment:
Yes, any ports the container listens on will be on the hosts network if net=host.  For the port mapping issue, that should be fixed in v0.3.1-rc2.  We are getting very close to releasing it.  Only one major blocker left.